packages/admin-ui-extensions/src/extension-api/LayoutApi/index.ts

```diff
@@ -4,15 +4,17 @@ export interface Layout {
   horizontal: SizeClass;
 }
 
+interface LayoutApiContent {
+  initialValue: Layout;
+  setOnChange: (onChange: (layout: Layout) => void) => void;
+}
+
 /**
  * You can use the Layout utility to adjust your content based on the user's screen size.
  * Layout returns `'compact'` on mobile devices and at narrow screen widths, and `'regular'` otherwise.
  */
 export interface LayoutApi {
-  layout: {
-    initialValue: Layout;
-    setOnChange: (onChange: (layout: Layout) => void) => void;
-  };
+  layout: LayoutApiContent;
```

Comment:
totally! there's a ticket for that already: https://github.com/Shopify/ui-extensions-private/issues/1613

this would involve changing the host implementation and introducing a breaking API change. this PR is just trying to get the docs script to work for APIs.